ssl.go

```diff
@@ -156,24 +158,39 @@ func sslCertificateAuthority(tlsConf *tls.Config, o values) error {
 	//
 	// https://github.com/postgres/postgres/blob/REL9_6_2/src/interfaces/libpq/fe-secure-openssl.c#L950-L951
 	if sslrootcert := o["sslrootcert"]; len(sslrootcert) > 0 {
-		tlsConf.RootCAs = x509.NewCertPool()
-
-		sslinline := o["sslinline"]
+		var caCertPool *x509.CertPool
 
-		var cert []byte
-		if sslinline == "true" {
-			cert = []byte(sslrootcert)
-		} else {
+		if sslrootcert == "system" {
 			var err error
-			cert, err = ioutil.ReadFile(sslrootcert)
+
+			caCertPool, err = x509.SystemCertPool()
 			if err != nil {
 				return err
 			}
-		}
 
-		if !tlsConf.RootCAs.AppendCertsFromPEM(cert) {
-			return fmterrorf("couldn't parse pem in sslrootcert")
+		} else {
+			caCertPool = x509.NewCertPool()
+
+			sslinline := o["sslinline"]
+
+			var cert []byte
+			if sslinline == "true" {
+				cert = []byte(sslrootcert)
+			} else {
+				var err error
+				cert, err = ioutil.ReadFile(sslrootcert)
+				if err != nil {
+					return err
+				}
+			}
+
+			if caCertPool.AppendCertsFromPEM(cert) {
```

Comment:
```suggestion
			if !caCertPool.AppendCertsFromPEM(cert) {
```
ref: https://pkg.go.dev/crypto/x509#CertPool.AppendCertsFromPEM returns `bool ok` so false is an error.

Reply:
Taken care of.